src/printer.cpp

```diff
@@ -172,17 +176,21 @@ void buildMaps(const ComponentEntityPtr &componentEntity, ComponentMap &componen
     }
 }
 
-std::string Printer::PrinterImpl::printUnits(const UnitsPtr &units) const
+std::string Printer::PrinterImpl::printUnits(const UnitsPtr &units, IdList &idList, const bool autoIds)
```

Comment:
Don't need `const bool` to basic a type to bother with constness.

Reply:
23a39d46